core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -49,7 +48,7 @@ THE SOFTWARE.
     </div>
 
     <j:forEach var="m" items="${it.managementLinks}">
-      <l:hasPermission permission="${m.requiredPermission}">
```

Comment:
see comment about accessCOntrolled and being able to have multiple permissions.
similar could be used in the layout taglib  `l:hasAnyPermission persionnsion="m.requiredPermissions"`?